apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/ServerRequiredTest.java

```diff
@@ -45,6 +45,9 @@ public class ServerRequiredTest {
   @GuardedBy("class synchronized")
   protected static RequestSpecification requestAuth;
 
+  /** The Server test utils instance, for convenience and brevity. */
+  protected ServerTestUtils testUtils = ServerTestUtils.get();
```

Comment:
Added this for some convenience, instead of needing a lengthy ServerTestUtils.get() everywhere its used.